build_all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+# build initrd
+sudo ansible-playbook -i hosts common/initrd_wrapper.yml -e "config_file=vars/initrd.yml provisioner=roles/initrd/provision_initrd"
```

Comment:
Do you think it would be clearer to line break here as you have done in the README example?